spiking_neurons/spiking_neuron.jl

```diff
@@ -0,0 +1,58 @@
+using Revise, MathematicalSystems,Reachability, LinearAlgebra, HybridSystems
+using LazySets;
+using Reachability: solve
+using Plots
+using TaylorIntegration
+
+@taylorize function spiking_neuron!(t, x, dx)
+    local a = 0.02
+    local b = 0.2
+    local I = 40
+    dx[1] = 0.04*x[1]^2 + 5*x[1] + 140 - x[2] + I
+    dx[2] = a*(b*x[1] - x[2])
+end
+
+function spiking_neurons()
+
+    automaton = LightAutomaton(1) # one node
+
+    inv_one = HPolyhedron([HalfSpace([-1.0, 0.0], -0.2),    #x[1] >= -0.2
```

Comment:
```suggestion
    inv_one = HPolyhedron([HalfSpace([-1.0, 0.0], 0.2),    #x[1] >= -0.2
```
The constraints in this file are wrong. `ax <= b` is equivalent to `-ax >= -b`.

Reply:
Thanks, now what remains to plot x[1] vs t and [2] vs t graphs.